### DISCUSSION TITLE
File transfer from ollie to mistral/levante not working

### DISCUSSION THREAD

**Discussion by fernandadialzira** [esm-tools/esm_tools#610] [HPCs] [answered]
I have been trying to transfer some files I have on ollie to mistral but I somehow it does not work. I also tried copying from ollie to levante. However, copying files from mistral to ollie works just fine.

Commands:
1. `scp -r fmatos@ollie1.awi.de:/work/ollie/fmatos/rho-amoc/notebooks/ ./`
message: 
```
ssh: connect to host ollie1.awi.de port 22: Connection timed out
```
2. `rsync -avz fmatos@ollie1.awi.de:/work/ollie/fmatos/rho-amoc/notebooks/ ./`
message:
```
ssh: connect to host ollie1.awi.de port 22: Connection timed out
rsync: connection unexpectedly closed (0 bytes received so far) [receiver]
rsync error: unexplained error (code 255) at io.c(600) [receiver=3.0.6]
```
I also tried using ollie0, but it still would not transfer any file. Did I do something wrong on the commands, or is this a common issue, or perhaps there is a "right" way to do it?

**Comment by denizural** [comment, marked as answer]
Hi @fernandadialzira, 
To connect Ollie you need an AWI VPN connection. What I recommend is from Ollie start a SFTP connection to Mistral and try to `put` / `get` files there. It should be OK to send / receive data from Ollie (eg. you are issuing the commands on Ollie). Could you please try and see if it works?
Alternatively, you can also use `scp` from Ollie

**Reply by fernandadialzira** [reply]
I am on a VPN connection actually, but I tried the `scp` from Ollie and it worked. I have completely forgotten to try the reverse way. Still, it is a bit weird that a normal `scp` or `rsync` from mistral on an AWI VPN connection did not work...

**Reply by seb-wahl** [reply]
Well mistral doesn't know whether you are on VPN or not. Since mistral is not part of the AWI network it will never work copying files to/from mistral to ollie. Only copying to/from ollie to mistral will work.
